src/pages/VideoFavDetail/VideoFavDetail.page.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
```

Comment:
This is fine like this, but just as future recommenation, since this view is a special case of `VideoDetail`, a workaround can be made to avoid duplicating code, having a parent view which `VideoFavDetail` will extend or validate in `VideoDetail` the necessary to handle both cases.